eth/tracers/api.go

```diff
@@ -642,6 +659,97 @@ func (api *API) traceBlock(ctx context.Context, block *types.Block, config *Trac
 	return results, nil
 }
 
+// traceInternalsAndAccounts configures a new tracer according to the provided configuration, and
+// executes all the transactions contained within. The return value will be one item
+// per transaction, dependent on the requestd tracer, and all dirty accounts.
+func (api *API) traceInternalsAndAccounts(ctx context.Context, block *types.Block, config *TraceConfig) (*internalAndAccountResult, error) {
+	if block.NumberU64() == 0 {
```

Comment:
nit: I think we can create a helper function to be used by both `traceBlock` and `traceInternalsAndAccounts` to reduce code duplication

Reply:
I quite hard since `traceBlock` returns an array but `traceInternalsAndAccounts` returns a struct